src/main/java/study/step2/domain/LottoRank.java

```diff
@@ -0,0 +1,34 @@
+package study.step2.domain;
+
+import java.util.Arrays;
+
+public enum LottoRank {
+    MISS(0, 0),
+    FOURTH(3, 5000),
+    THIRD(4, 50000),
+    SECOND(5, 1500000),
+    FIRST(6, 2000000000);
```

Comment:
Enum 잘 구현해주셨네요 👍 

아래처럼 메시지와 함께 써도 좋겠네요!
```suggestion
public enum LottoRank {
    MISS(0, 0, "꽝"),
    FOURTH(3, 5_000, "3개 일치"),
    THIRD(4, 50_000, "4개 일치"),
    SECOND(5, 1_500_000, "5개 일치"),
    FIRST(6, 2_000_000_000, "6개 일치");
```